doc/source/documentation_style/docstrings.rst

```diff
@@ -24,14 +29,45 @@ PyAnsys library docstrings contain at a minimum the following
 * `Returns <https://numpydoc.readthedocs.io/en/latest/format.html#returns>`_ if applicable
 * `Examples <https://numpydoc.readthedocs.io/en/latest/format.html#examples>`_
 
-These sections should follow the numpydoc standards. To avoid
+These sections should follow numpydoc standards. To avoid
 inconsistencies between PyAnsys libraries, adhere to the additional 
 standards that follow.
 
+Classes
+~~~~~~~
+A class is a "noun" representing a collection of methods. For consistency within PyAnsys libraries,
+always start the short description for a class with a verb ending in "s", followed by an extended
+summary if applicable::
+
+  class FieldAnalysis3D(Analysis, object):
```

Comment:
Explicitly deriving from  ``object`` is no longer needed in Python 3. I think this line should just read ``class FieldAnalysis3D(Analysis):``